s/setprototypeof/setprototypeof_v1.1.1_rhel_8.5.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : setprototypeof
+# Version           : v1.1.1
```

Comment:
is the script specific to this version or will work for top-of-tree as well? If its the latter, can we remove version from script name to make it generic?

Reply:
renamed the file as suggested.